src/fragments/chatcontent.tsx

```diff
@@ -4,10 +4,11 @@ import { SingleSignOnEIP4361AuthProvider, USER_ADDRESS_PARAM_EXTERNAL_EIP4361 }
 import { decodeB64 } from "../../utils/common";
 import { useCeramicContext } from "../../context";
 import { decryptWithTACo, parseUrsulaError } from "../../utils/taco";
-import { authenticateCeramic, alreadyLoggedIn, getCeramicSiweInfo } from "../../utils";
+import { authenticateCeramic, alreadyLoggedIn, getCeramicSiweInfo, reset } from "../../utils";
```

Comment:
Does `reset` still need to be imported?

Reply:
thanks!
I've got used to pre commit hooks so forgot to check by myself what to clean.
✔️